backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -109,6 +109,11 @@ def _deserialize(self, value, attr, data, **kwargs):
         return None
 
 
+ExcludeAuthorIdField = marshmallow.fields.Int(
+    example=10, validate=positive_int_validator, missing=0, allow_none=False,
```

Comment:
Same comment here.

Reply:
BTW the naming is too restrictive as this field type is not reserved to author ids. Maybe something like `PositiveInteger`?

Reply:
Indeed. Anyway, this is going to be changed to a list of positive integer. Are you fine with PosivieIntegerListField?